stdlib/2/SocketServer.pyi

```diff
@@ -54,12 +54,12 @@ class UDPServer(BaseServer):
 
 if sys.platform != 'win32':
     class UnixStreamServer(BaseServer):
-        def __init__(self, server_address: Tuple[str, int],
+        def __init__(self, server_address: AnyStr,
                      RequestHandlerClass: type,
                      bind_and_activate: bool = ...) -> None: ...
 
     class UnixDatagramServer(BaseServer):
-        def __init__(self, server_address: Tuple[str, int],
+        def __init__(self, server_address: AnyStr,
```

Comment:
Per my comment in the bug report, these should be `Union[Text, bytes]`. `AnyStr` is a generic, not an alias.

Reply:
My understanding was, that it only matters when there is more than one occurrence of `AnyStr`, because then you can't mix `bytes` and `str`. What am i missing? Will the server type then be a generic type parametrized on the string type?

Reply:
@nielsbuwen Using `AnyStr` as shorthand for a `Union` can have unexpected effects. For example the following does not type check:

```python
from typing import AnyStr, Union

def x(y: AnyStr) -> None:
    pass

z: Union[bytes, str]
x(z)
```